code/modules/research/server.dm

```diff
@@ -103,30 +121,9 @@
 /obj/machinery/rnd/server/proc/get_env_temp()
 	var/turf/L = loc
 	if(isturf(L))
-		return L.temperature
+		return L.return_temperature()
 	return 0
 
-/obj/machinery/rnd/server/proc/produce_heat(heat_amt)
```

Comment:
This was never called before so you're adding new functionality and it should be in the changelog :trollface: